sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -754,4 +754,29 @@ class PlanParserSuite extends AnalysisTest {
       assertEqual(query2, Distinct(a.union(b)).except(c.intersect(d, isAll = true), isAll = true))
     }
   }
+
+  test("create/alter view as insert into table") {
+    intercept[ParseException](parsePlan("CREATE VIEW testView AS INSERT INTO jt VALUES(1, 1)"))
+    val sql1 =
+      """
+        |CREATE VIEW testView AS FROM jt
+        |INSERT INTO tbl1 SELECT * WHERE jt.id < 5 " +
```

Comment:
`INSERT INTO tbl1 SELECT * WHERE jt.id < 5 " + INSERT INTO tbl2 SELECT * WHERE jt.id > 4`

what are we testing for it?

Reply:
@cloud-fan oops.. will fix.